tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -509,3 +507,15 @@ Other free text and markdown formatting can be used elsewhere in the document if
 
 ### [shares are not listed with full paths](https://github.com/owncloud/ocis/issues/2462)
 -   [webUISharingPublicBasic/publicLinkCreate.feature:88](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature#L88)
+
+### [web config update is not properly reflected after the ocis start](https://github.com/owncloud/ocis/issues/2944)
+-   [webUIFiles/breadcrumb.feature:50](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFiles/breadcrumb.feature#L50)
+
+### Shares folder cannot be deleted and is always visible
+-  [webUIDeleteFilesFolders/deleteFilesFolders.feature:77](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L77)
+-  [webUIDeleteFilesFolders/deleteFilesFolders.feature:89](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature#L89)
+-  [webUISharingAcceptShares/acceptShares.feature:72](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L72)
+-  [webUIOperationsWithFolderShares/accessToSharesFolder.feature:14](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIOperationsWithFolderShares/accessToSharesFolder.feature#L14)
```

Comment:
this line will be gone from the expected to fail file in ocis after https://github.com/owncloud/ocis/pull/3007 gets merged, so also can be deleted here after cherry picking commit from #6294

Reply:
This still needs to be done, right?

Reply:
yes, this PR needs the oCIS commit updated to whatever is latest, and the expected-failures adjusted...

Reply:
ok, thanks. Please ping me again when that's done. Or if you want to have this merged as an intermediate state then I'm fine with that as well!

Reply:
Resolving this thread. I will ping again in the overall PR comments when I think this is ready.